sunpy/instr/fermi.py

```diff
@@ -40,59 +40,39 @@ def download_weekly_pointing_file(date):
     # use a temp directory to hold the file
     tmp_dir = tempfile.mkdtemp()
     # use Fermi data server to access weekly LAT pointing file.
-    base_url = 'http://fermi.gsfc.nasa.gov/ssc/observations/timeline/ft2/files/'
-    fbasename='FERMI_POINTING_FINAL_'
-
+    base_url = 'ftp://legacy.gsfc.nasa.gov/FTP/glast/data/lat/weekly/spacecraft/'
+    fbasename = 'lat_spacecraft_weekly_w'
+    
     # find out which file to get based on date
-    # earliest file in the FERMI server is for mission week 23,
-    # beginning 2008 November 6.
-    weekly_file_start=parse_time('2008-11-06')
-    base_week=23
+    # earliest full file in the FERMI server is for mission week 10,
+    # beginning 2008 August 7.
+    weekly_file_start=parse_time('2008-08-07')
+    base_week=10
```

Comment:
Style only - please ensure that there is one space on either side of the equal signs (no spaces when defining function keywords).